### ISSUE TITLE
Editing contact birthday with omitting year is buggy

### ISSUE THREAD

**Issue by provokateurin** [nextcloud/contacts#4179]
### Steps to reproduce

1. Create a contact with a birthday and tick the "Omit year" checkbox, save it and reload the page
2. Edit the contact, untick the "Omit year" checkbox, select a date for the birthday

### Expected behavior

The year should be displayed and the checkbox should stay unticked.

### Actual behavior

The year is hidden and the checkbox is ticked again.

### Contact version

5.5.3

### Operating system

Alpine

### PHP engine version

PHP 8.2

### Web server

Nginx

### Database

PostgreSQL

### Additional info

As a workaround one can untick the box, save, edit the date and save again.

**Comment by GVodyanov** [comment]
Hey! After editing the contact in step two, do you save and reload the page, and then still not see the year? I don't think I'm able to reproduce what you're describing

**Comment by provokateurin** [comment]
No, between editing the date and unticking the box I do not save and reload.

**Comment by DerDreschner** [comment]
Hey @provokateurin , thanks for your report! I tried to reproduce it right now and I'm unable to do so. Could you retest again and tell me if that's the case for you as well? Thanks!!

**Comment by provokateurin** [comment]
No, the issue is gone now, however when ticking the "Omit year" checkbox, the year is now replaced with 1900 and displayed as such 🙈 This is on latest master for server and contacts btw.

**Comment by DerDreschner** [comment]
Yeah, that's the same behavior I experienced. Which should be fine as default value I in this case (I guess).

I'll close the issue. If you think otherwise and we should change the current behavior, just let me know or open a new issue! Thanks! 😄

**Comment by provokateurin** [comment]
I know that in the past no year was displayed and this was much better than showing 1900 to the user.

**Comment by DerDreschner** [comment]
Do you mind opening a new ticket for that then, @provokateurin? If it's been there before then it should be applied again.

**Comment by provokateurin** [comment]
https://github.com/nextcloud/contacts/issues/5132
